documentation/API_Release_Guidelines.md

```diff
@@ -90,66 +94,75 @@ Example of continuous release numbering of an API version across its release typ
 | Release type | API version | release tag | release package | release package label |
 |------|------|:------:|:------:|:------:| 
 | N/A | work-in-progress | N/A | N/A | N/A |
-| pre-release | alpha | rx.1 ... rx.m | optional | optional: "pre-release" |
-| pre-release | release-candidate | rx.m+1 ... rx.n | mandatory | "pre-release" |
-| release | public | rx.n+1 | mandatory | "latest" |
-| maintenance release | public | rx.n+2 ... rx.n+p | mandatory | "latest" |
+| pre-release | alpha | rx.1 ... rx.m | optional | optional: `pre-release` |
```

Comment:
I would delete "optional" in the last column of an alpha release ... the release is optional, but if done it is always a pre-release:

```suggestion
| pre-release | alpha | rx.1 ... rx.m | optional | `pre-release` |
```

Reply:
makes sense, done